spec/src/main/asciidoc/metrics.adoc

```diff
@@ -20,17 +20,46 @@
 [[metrics]]
 == Metrics
 
+Metrics are captured measurements of applications' and runtime's behavior.
+An application may provide metrics of its own in addition to the metrics provided by the runtime.
 
-=== Access to the OpenTelemetry Metrics API
-An implementation of MicroProfile Telemetry Metrics MUST provide the following CDI beans for supporting contextual instance injection:
+Custom metrics can be defined by utilizing <<metrics_api,Metrics API>> and implementations are required to capture <<required_metrics,required metrics>>.
 
-* `io.opentelemetry.api.metrics.Meter`
+The following example demonstrates how to define a custom metric using the Metrics API:
 
+[source,java]
+----
+class WithCounter {
+    @Inject
+    Meter meter;
 
+    private LongCounter counter;
 
+    @PostConstruct
+    public void init() {
+        LongCounter longCounter =
+                meter
+                    .longCounterBuilder("new_subscriptions")
+                    .setDescription("Number of new subscriptions")
+                    .setUnit("1")
+                    .build();
+    }
 
+    void subscribe(String plan) {
+        counter.add(1,
+            Attributes.builder().put("plan", plan).build());
+    }
+}
+----
+`Meter` is used to define an instrument, in this case a Counter and application code then can record measurements with additional attributes.
 
+[#metrics_api]
+=== Access to the OpenTelemetry Metrics API
```

Comment:
I suggest you move this above the custom metrics definition as in your example you refer to the injection of `Meter`

Reply:
I was aiming for small motivation paragraph before we get to detailed specification. Now that all of Don's changes are merged in it is more apprarent.